### DISCUSSION TITLE
Adding Element for annotation or button feature to Spoke

### DISCUSSION THREAD

**Discussion by profhupe** [mozilla/hubs#5594] [Spoke Questions]
I was wondering if anyone has developed an element in Spoke that would allow someone to annotate an environment with hotspots that could be clicked on to reveal additional text, video, audio, or images. You could then provide additional information or create a guided tour of environments without cluttering the overall scene. Has this been discussed or done already? 

Thanks!

**Comment by matthewbcool** [comment]
Would require significant development and design work, but possible with Hubs Cloud.

Paradowski Creative has developed some custom components along these lines: [Apart Poster Gallery](https://show.apartposters.com/ssComLt/showroom-restrospective-2) [Quick Screen Recording](
https://gyazo.com/78376fbdfad78911009f0eeb0fbd6f55)

There may be creative ways for you to achieve results close to what you are looking for with the existing constraints by leveraging different components. Might be helpful to take a look at some ways others have set up spaces and used different components. There is a lot of design opportunity we have yet to explore in Hubs. 

https://hubs.mozilla.com/labs/exhibitions-that-educate/
https://hubs.mozilla.com/labs/5-incredible-art-galleries/
https://hubs.mozilla.com/labs/what-is-the-blender-add-on/
